QPong-Unity-Version/Assets/Prefabs/Gate.prefab

```diff
@@ -0,0 +1,207 @@
+%YAML 1.1
```

Comment:
I didn't know prefab is used in this way. I put the prefabs on scenes... I will do the same for the paddle.